projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts

```diff
@@ -221,6 +226,7 @@ export class WorkAllocationService {
       tasks: []
     };
     if (!this.isWAEnabled()) {
+      this.appConfig.logMessage(`search for completable call missed for ${caseId} in event ${eventId}`);
```

Comment:
Can you change this message to "isWAEnabled false for ${caseId} in event {eventId}"